### ISSUE TITLE
Crash related to name reading on returnUserList

### ISSUE THREAD

**Issue by xseano** [xseano/xQube#3]
Issue may be related to new client's name being set asynchronously after a returnUserList packet is sent out to other clients? Still exploring this issue.

```
# ../xqube/server/src/xclass/Block.js:147
          for (var y = 0; y < this.xQube.userList[t].name.length; y++) {
                                                         ^

TypeError: Cannot read property 'length' of undefined
```

**Comment by IceAquaria** [comment]
Were you able to fix this issue?

**Comment by xseano** [comment]
Yes, this was fixed at [ce4d3fa](https://github.com/xseano/xQube/commit/ce4d3fa0522966d5de951b990df0c6c0f18b8a2f).
